src/odemis/acq/stitching/test/weaver_test.py

```diff
@@ -189,6 +192,33 @@ def test_synthetic_perfect_overlap(self):
 
         numpy.testing.assert_equal(outd, exp_out)
 
+    def test_rotated_tiles(self):
+        """Verify that the correct rotation is set on the weaved image."""
+        numTiles = [2, 3, 4]
+        overlap = 0.4
+        rotation = math.radians(5)
+        if self.weaver_type == WEAVER_COLLAGE:
+            weaver_class = CollageWeaver
+        elif self.weaver_type == WEAVER_COLLAGE_REVERSE:
+            weaver_class = CollageWeaverReverse
+        elif self.weaver_type == WEAVER_MEAN:
+            weaver_class = MeanWeaver
+
+        for img in IMGS:
+            conv = find_fittest_converter(img)
+            img = conv.read_data(img)[0]
+            img = ensure2DImage(img)
+            for n in numTiles:
+                [tiles, _] = decompose_image(img, overlap, n, "horizontalZigzag", False)
```

Comment:
Could you indicate somewhere that the content of the image will be bad looking, because of the rotation is not taken into account when decomposing the image (and that doesn't matter because we don't check for the image content).